### PR TITLE
Tweaks git checkout UI output

### PR DESCRIPTION
**Pull request by jordanthomas**
Tweaking the "Updating ..." output of the checkout method to reduce
confusion when running `outdated`. Opting instead for "Fetching" as it
does when doing a clone operation.

### CODE REVIEW

File: lib/bundler/source/git/git_proxy.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def version
         def checkout
           if path.exist?
             return if has_revision_cached?
-            Bundler.ui.confirm "Updating #{uri}"
+            Bundler.ui.info "Fetching #{uri}"
```
Comment (segiddins) [inline lib/bundler/source/git/git_proxy.rb]: Why the change from `confirm` to `info` here?

Comment (jordanthomas) [inline lib/bundler/source/git/git_proxy.rb]: The colored output of `confirm` looked out of place next to the other "Fetching" outputs. It was also unclear to me what the coloring was trying to convey when the same method doesn't use `confirm` when cloning.

Picture from my machine for reference:
bundler

Comment (indirect) [inline lib/bundler/source/git/git_proxy.rb]: I agree that git updates should be info rather than confirm—in the `install` output, confirm means a gem is being installed, while info means a gem is already installed and is simply being used.

Comment (segiddins) [inline lib/bundler/source/git/git_proxy.rb]: :+1:

**Comment by indirect** [comment]
@homu r+
